internal/store/job.go

```diff
@@ -302,6 +302,35 @@ var (
 					}
 				}
 
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		},
+		{
+			Name: "kube_job_status_failed_reason",
```

Comment:
I am curious if we should instead add the reason label to the already existing metric `kube_job_status_failed ` cc @brancz @tariq1890 any preferences?

Reply:
I was also thinking on the similar lines. @lilic, @brancz & @tariq1890 - Should I make the changes accordingly?

Reply:
yeah let's collapse it into one I agree

Reply:
Cool, I'll take care of it. 👍